app/models/avenger.rb

```diff
@@ -1,3 +1,6 @@
 class Avenger < ApplicationRecord
   belongs_to :user
+  validates :name, presence: true
```

Comment:
I think it'd be better if there is the name is unique so there is not several captain america for ex 🙂